packages/api-explorer/__tests__/ResponseBody.test.jsx

```diff
@@ -170,8 +170,19 @@ describe('Response body', () => {
   });
 
   test('should display Result if result.responseBody is set', () => {
-    const responseBody = mount(<IntlProvider><ResponseBody {...oauthInvalidResponse} oas={oas} /></IntlProvider>);
-    expect(responseBody.find(Result)).toHaveLength(1);
+    const responseBody = mount(<IntlProvider><ResponseBody
```

Comment:
Can you do a different test for 0 and for undefined?

Reply:
Sure!

Reply:
I found out we need only to check 0 and null, since those are the only options returned by the response client